cts/cts-cli.in

```diff
@@ -1309,8 +1309,25 @@ function test_tools() {
     cmd="crm_resource -a -r gr2 --force"
     test_assert $CRM_EX_OK 0
 
+    # Create a shadow CIB based on constraints.xml
+    create_shadow_cib --create
```

Comment:
This is handy, and I believe there are other opportunities to use this in `cts-cli`, where we currently make copies manually. I'll try to remember this going forward.